src/ucs/sys/compiler_def.h

```diff
@@ -28,7 +28,7 @@
      switch(0) {case 0:case (_cond):;}
 
 /* Maximal allocation size for on-stack buffers */
-#define UCS_ALLOCA_MAX_SIZE  1200
+#define UCS_ALLOCA_MAX_SIZE  4096
```

Comment:
why are allocation max size needed?

Reply:
Since protos priv buffer is allocated on stack, and after increasing maximal nubmer of lanes some of priv buffers started to exceed the alloca max size. But anyway that doesn't matter now, since we decided to move priv allocation to the heap.